components/extractors/other/telemetry-data/telemetry-data.md

```diff
@@ -163,6 +163,8 @@ and their consumption metrics.
 | `time_credits_used` | Number of time credits consumed by the workspace on the particular date | `2.413333` |
 | `billed_credits_used` | Number of the actually billed credits | `2.413333` |
 
+***Note:** A value of `0` in `data_app_runtime_hours`,`time_credits_used` or `billed_credits_used` means a correction was made. The row couldn’t be removed due to incremental downloads, so the value was cleared instead.*
```

Comment:
`incremental downloads` bych změnil na `incremental loads`, což je termín, který používáme v našich docs.
`the value was cleared` bych změnil na `the original value was set to zero`, protože "cleared" zní spíš jako `null` či `empty`. A pak bych jen vypíchl, že měnimíe původní (original) hodnotu.